content/en/docs/reference/node/seccomp.md

```diff
@@ -0,0 +1,117 @@
+---
+content_type: reference
+title: Seccomp and Kubernetes
+weight: 80
+---
+
+<!-- overview -->
+
+Seccomp stands for secure computing mode and has been a feature of the Linux
+kernel since version 2.6.12. It can be used to sandbox the privileges of a
+process, restricting the calls it is able to make from userspace into the
+kernel. Kubernetes lets you automatically apply seccomp profiles loaded onto a
+{{< glossary_tooltip text="node" term_id="node" >}} to your Pods and containers.
+
+## Seccomp fields
+
+{{< feature-state for_k8s_version="v1.19" state="stable" >}}
+
+There are three ways to specify a seccomp profile for a
+{{< glossary_tooltip text="pod" term_id="pod" >}}:
+
+- for the whole Pod using [`spec.securityContext.seccompProfile`](/docs/reference/kubernetes-api/workload-resources/pod-v1/#security-context)
+- for a single container using [`spec.containers[*].securityContext.seccompProfile`](/docs/reference/kubernetes-api/workload-resources/pod-v1/#security-context-1)
+- for an [ephermal container](/docs/concepts/workloads/pods/ephemeral-containers) using [`spec.ephemeralContainers[*].securityContext.seccompProfile`](/docs/reference/kubernetes-api/workload-resources/pod-v1/#security-context-2)
```

Comment:
what about init containers and restartable init containers?

Reply:
Oh yeah, added them as well.